lib/berkshelf/source.rb

```diff
@@ -53,7 +53,7 @@ def search(name)
       universe
         .select { |cookbook| cookbook.name =~ Regexp.new(name) }
         .group_by(&:name)
-        .collect { |name, versions| versions.max_by(&:version) }
+        .collect { |name, versions| versions.max_by { |v| Semverse::Version.new(v.version) } }
```

Comment:
Hrm, sooooo this is a _really_ expensive operation. Parsing a semver version was one of the larger bottlenecks in our previous Ruby-based depsolver.

I think, if we do this, we will need to implement some comparisons using bit-shifts in semverse to make this feasible. With hundreds of cookbooks, you could be approaching a minute to sort :bullettrain_side: